src/main/resources/META-INF/frontend/fc-xterm/xterm-element.ts

```
@@ -161,6 +161,8 @@ export class XTermElement extends LitElement implements TerminalMixin {
   bellStyle: 'none' | 'sound'
   
   customKeyEventHandlers: CustomKeyEventHandlerRegistry;
+  
+  addons : Object = {};
```

Biome (1.9.4)

[error] Don't use 'Object' as a type.

Prefer explicitly define the object shape. This type means "any non-nullable value", which is slightly better than 'unknown', but it's still a broad type.

(lint/complexity/noBannedTypes)